mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -264,6 +309,64 @@ def test_transpiled_with_export_to_xlsx_format_rows_no_header():
 ])
 """
 
+
+# This tests when the user exports a dataframe with row formatting without header formatting.
+def test_transpiled_with_export_to_xlsx_conditional_format():
+    df = pd.DataFrame({'A': [1, 2, 3]})
+    mito = create_mito_wrapper(df, arg_names=['df'])
+    mito.set_dataframe_format(0, DF_FORMAT_CONDITIONAL)
+    filename = 'test_format_conditional.xlsx'
+    mito.export_to_file('excel', [0], filename)
+    assert "\n".join(mito.transpiled_code) == """from mitosheet.public.v3 import *
```

Comment:
Currently, we're testing that the generated code is correct. This is an OK first step -- but it doesn't test the real thing we want to, which is that the cell color in the Excel is correct. 

I asked ChatGPT to get the cell color from an Excel file - it gave me this:
```
import pandas as pd
from openpyxl import load_workbook

# Step 1: Read the Excel file using Pandas
file_path = 'path/to/your/excel_file.xlsx'
df = pd.read_excel(file_path, sheet_name='Sheet1')

# Step 2: Load the workbook using openpyxl
wb = load_workbook(file_path)

# Step 3: Get the specific cell whose background color you want to read
sheet_name = 'Sheet1'
cell_address = 'A1'  # Replace this with the desired cell address

sheet = wb[sheet_name]
cell = sheet[cell_address]

# Step 4: Access the background color of the cell
background_color = cell.fill.start_color.index

# Step 5: Convert the color index to the RGB format
# The background_color variable will be something like 'FFFF0000'
# where the first two characters represent transparency, and the last six characters represent the RGB color.
rgb_color = background_color[2:]
# If you need a "#" before the RGB value (e.g., "#FF0000"), you can add it like this:
# rgb_color = "#" + rgb_color

# Step 6: Now you can use the RGB color value as needed
print("Background color of cell {} is: {}".format(cell_address, rgb_color))
```

Want to try writing a helper function of the type `get_cell_background_color(file_name, sheet, row, column) -> str` -- and then we can use it in tests to make sure that we're getting correct values? 

I think it's worth investing in some good testing infrastructure before we scale to the rest of the conditional format conditions.

Reply:
Ok sounds good!